src/ukify/ukify.py

```diff
@@ -661,7 +661,7 @@ def make_uki(opts):
         ('.splash',  opts.splash,     True ),
         ('.pcrpkey', pcrpkey,         True ),
         ('.initrd',  initrd,          True ),
-        ('.uname',   opts.uname,      False),
+        ('.uname',   opts.uname,      True ),
```

Comment:
please move this up too, now that this section isn't special anymore. 

i.e. stick to the same order everywhere please